src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -43,7 +43,16 @@ public ArrayList<Integer> getFavouriteNumbers() {
         TODO: 1. Create a method named getSecondNumber that returns a whole number. It must return the
            second number contained in the list that is returned from getFavouriteNumbers
      */
-
+    public int getSecondNumber(){
+        ArrayList<Integer> arr = getFavouriteNumbers();
+        int tall=0;
+        for (int i = 0; i<arr.size(); i++){
+            if (i==1){
+                tall= arr.get(i);
+            }
+        }
+        return tall;
+    }
```

Comment:
Could be simplified, eg:

```
public int getSecondNumber() {
    ArrayList<Integer> arr = getFavouriteNumbers();
    if (arr.size() > 1) {
        return arr.get(1);
    } else {
        // Handle the case where the array does not have a second element
        // This could be returning a default value or throwing an exception
        return 0; // or throw new IllegalStateException("List does not have enough elements");
    }
}
```